kong/plugins/influxdb-log/handler.lua

```diff
@@ -0,0 +1,105 @@
+local HttpLogHandler = require "kong.plugins.http-log.handler"
+
+local InfluxdbLogHandler = HttpLogHandler:extend()
```

Comment:
Yes, same. I made my point about logging plugins in [this comment](https://github.com/Mashape/kong/pull/1731#issuecomment-252642716) (and thread): I am very much against inter-plugins dependencies very much (especially considering the `:extend()` method is bound to disappear, as plugins should not be inheritable, this is just a side effect of using `basic_plugin.lua` which uses `classic.lua` (old legacy code).

Instead, and that is the reason why `log_serializers` exist, there should be a serializer for each service (as well as, optionally, a buffer - responsible to send the data). In this scenario the only logging plugins that should stay a per-protocol: `http`, `tcp`, `udp`, and can come with facilities such as a config field for the serializer and buffer.